package.json

```diff
@@ -135,8 +135,34 @@
       "react/display-name": [
         0
       ],
-      "react/forbid-component-props": 0,
-      "react/forbid-dom-props": 0
+      "react/forbid-component-props": [
+        2,
+        {
+          "forbid": [
+            {
+              "propName": "style",
+              "disallowedFor": [
```

Comment:
hmmm this seems counter intuitive to the purpose of this PR. shouldn't these rules apply everywhere?

Reply:
If we remove `disallowedFor`, it will be apply everywhere.